ISSUE_TEMPLATE/NEW-TASK.yaml

```diff
@@ -0,0 +1,50 @@
+name: New Task
+description: New task
+title: "[Task]: "
+labels: []
+type: task
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Describe the task
+      placeholder: What needs to be done?
+      value: "A task needs to be done!"
```

Comment:
If I read this correct, the field will have this value pre-filled. Is that necessary when we have a placeholder?